python/tests/core/test_transformation_function_engine.py

```diff
@@ -241,6 +271,7 @@ def test_attach_transformation_fn_td(self, mocker):
         feature_store_id = 99
```

Comment:
Some test cases to see if it can attach tf with prefix and other cases would be nice.